components/layout.js

```diff
@@ -0,0 +1,52 @@
+import Link from 'next/link'
+import TwitterLogo from './twitter.svg'
+import DiscordLogo from './discord.svg'
+import TelegramLogo from './telegram.svg'
+import Image from 'next/image'
+console.log(TwitterLogo)
```

Comment:
do we want to check in console.log(TwitterLogo)? lol

Reply:
oops, removing